src/app/shared/object-list/item-list-element/entity-types/orgunit/orgunit-list-element.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
```

Comment:
I think if we would like to have each entity as a "module", as something pluggable, I think we should group every specific code file inside the same tree structure...
Example: 
> typedItems
>> OrgUnit
>>> simple
>>> full
>>> object-list

>> Person